nyx_client/nyx_client/client.py

```diff
@@ -554,13 +555,14 @@ def create_data(
         size: int,
```

Comment:
As discussed:
1. `size` can now be optional
2. `size` should be populated from server
    - in response on data create/update, if possible.
    - when using `my_data_by_name` (and any others, if applicable)

Reply:
https://github.com/Iotic-Labs/nyx-sdk/pull/34/commits/2518b4320525e0fd8824cdb964113b232a89f317

Reply:
:+1: 
Still need setting of size from server responses in relevant places, unless I'm mistaken